--- src/netplan-0.99/Makefile   2020-04-15 22:33:27.000000000 +0000
+++ Makefile.modified   2020-08-12 16:26:10.162988140 +0000
@@ -131,14 +131,4 @@
 %.service: %.service.in
        sed -e "s#@ROOTLIBEXECDIR@#$(ROOTLIBEXECDIR)#" $< > $@

-
-%.html: %.md
-       pandoc -s --toc -o $@ $<
-
-doc/netplan.5: doc/manpage-header.md doc/netplan.md doc/manpage-footer.md
-       pandoc -s -o $@ $^
-
-%.8: %.md
-       pandoc -s -o $@ $^
-
 .PHONY: clean
