--- PKGBUILD    2020-08-12 16:43:47.045529405 +0000
+++ PKGBUILD.mod        2020-08-12 16:43:32.405631366 +0000
@@ -5,7 +5,7 @@
 pkgver=0.99
 pkgrel=1
 pkgdesc="Network configuration abstraction renderer"
-arch=('x86_64')
+arch=('aarch64')
 url="https://github.com/CanonicalLtd/netplan"
 license=('GPL3')
 depends=(
@@ -17,15 +17,19 @@
 makedepends=(
   'python'
   'pkgconf'
-  'pandoc'
 )
 source=(
   "https://github.com/CanonicalLtd/${pkgname}/archive/${pkgver}.tar.gz"
+  "netplan-Makefile.patch"
+)
+sha256sums=(
+  '6ed7e12c88f972361f794a715dbd80cb85b822d2dc8c4ca9c7605aebbc008cb3'
+  '380693f3c94244ca08c334c64e14f8eb822f11caa3c4c034f2b97ddd81f55510'
 )
-sha256sums=('6ed7e12c88f972361f794a715dbd80cb85b822d2dc8c4ca9c7605aebbc008cb3')

 build() {
   cd "${pkgname}-${pkgver}"
+  patch Makefile "${src}/netplan-Makefile.patch"
   env
   make
 }
